.travis.yml

```diff
@@ -1,10 +1,18 @@
 sudo: false
+
 language: go
 
 go:
-    - 1.2
+    - 1.3
     - tip
 
-install: go get -t ./...
+services:
+  - redis-server
 
-script: go test -v -short ./...
+install:
+  - make get-deps
+  # Move to the gopkg location that rather than the default clone location
+  - export GOPKG_DIR=$GOPATH/src/gopkg.in/throttled
+  - mkdir -p $GOPKG_DIR
+  - mv $TRAVIS_BUILD_DIR $GOPKG_DIR/throttled.v0
+  - cd $GOPKG_DIR/throttled.v0
```

Comment:
Ah, got it. Thanks.

On Mon, Aug 17, 2015 at 10:20 PM, Andrew Metcalf <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/throttled/throttled/pull/9#discussion_r37264945>:
>
> >
> > -script: go test -v -short ./...
> > +install:
> > +  - make get-deps
> > +  # Move to the gopkg location that rather than the default clone location
> > +  - export GOPKG_DIR=$GOPATH/src/gopkg.in/throttled
> > +  - mkdir -p $GOPKG_DIR
> > +  - mv $TRAVIS_BUILD_DIR $GOPKG_DIR/throttled.v0
> > +  - cd $GOPKG_DIR/throttled.v0
>
> If it finds a Makefile, Travis runs make as the test script by default.
> That's not obvious so I've updated the configuration to be explicit.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/throttled/throttled/pull/9/files#r37264945>.
>